contracts/drafts/dao/DAO.sol

```diff
@@ -0,0 +1,117 @@
+pragma solidity ^0.5.10;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20Mintable.sol";
+import "../../issuance/IssuanceEth.sol";
+import "../token/ERC20MultiDividendable.sol";
+
+
+contract DAO is ERC20Mintable, ERC20MultiDividendable, IssuanceEth {
+
+    mapping(address => uint256) public begAmount;
+    uint256 totalAmounts;
+    mapping(address =>
+        mapping(address => uint256)
+    ) public votesForIdeaByHolder;
+    mapping(address => uint256) public totalVotesForIdea;
+    mapping(address => uint256) public totalVotesByHolder;
+    mapping(address => address[]) public backersForIdea;
+    event Here();
+
+    constructor()
+    ERC20Mintable()
+    ERC20MultiDividendable()
+    IssuanceEth(address(this))
+    public
+    {
+        _createState("NEVER");
+        _createTransition("LIVE", "SETUP");
+        _createTransition("FAILED", "SETUP");
+        addMinter(address(this));
+    }
+
+    function transferFunds(address payable _wallet) public onlyOwner {
+        require(currentState == "NEVER", "You can never do this.");
+    }
+
+    function restartFundingRound() public onlyOwner {
```

Comment:
Let's leave this feature for more advanced contracts. In the base DAO.sol there is only one funding round (provided by inheritance from IssuanceEth.sol).

Reply:
Alright, this I can do, but I'd like to know why. Wouldn't there always be new members in the DAO (not just the founding members)?